web-platform-when-why/index.html

```diff
@@ -0,0 +1,423 @@
+            <aside class="notes">
+              Well, there are obvious reasons to not choose the web. For instance, if the platform does
+              not cover your API needs but, event when this is not a problem and having these both
+              interoperability and lack of friction features, so exclussive and so fetching the Web does
+              not seem to fit our needs. Why?
+            </aside>
+          </section>
+
+          <section>
+            <p><strong>User experience</strong></p>
+            <aside class="notes">
+              The answer is user experience. In the beginning, the Web was never designed for mobile usage.
+              We surfed the web by paying special attention to the browser, in desktop, powerful multicore
+              setups with big screens an a reliable network wired connection.
+            </aside>
+          </section>
+
+          <section>
+            <h3>Dependence on connectivity</h3>
+            <img alt="Dinosaur game in Chrome" src="./imgs/dinosaur-game-chrome.jpg" style="width: 70%;"/>
+            <aside class="notes">
+              In the mobile realm you can not rely on be always connected. If the connectivity is lost,
+              the experience is broken. The complete application vanishes. Of course a user can understand
+              an online application to not show content if offline but breaking with no information? I
+              don't think so.
+            </aside>
+          </section>
+
+          <section>
+            <h3>Dependendence on network speed</h3>
+            <img alt="Time a user can wait before leaving the page" src="./imgs/time-waiting-before-give-up.png" style="width: 50%"/>
+            <aside class="notes">
+              Not only the lack of connectivity is a problem but unreliable connection speeds are an issue
+              as well. Key point with native application is all about performance, and this is perceived
+              performance in the form of waiting time.
+            </aside>
+          </section>
+
+          <section>
+            <h3>Inability to re-engage the user</h3>
+            <img alt="Audience vs Loyalty" src="./imgs/audience-vs-loyalty.png"/>
+            <aside class="notes">
+            </aside>
+          </section>
+
+          <section>
+            <h3>Performance</h3>
+            <img alt="Causes making milennia to give up an application" src="./imgs/give-up-causes-milennia.png"/>
+            <p>
+              And last but not least, there is more about perceived performance. Interoperability
+              comes with a cost on translation overhead and this reduces the effective speed of the
+              technology. In the Web, we need to be extra careful to keep things flowing. Given the
+              single thread nature of JavaScript, if a computation takes so long, frames per second
+              will drop under 60 fps giving the sensation of slow and junked animations.
```

Comment:
Consider changing "junked" for "jerky" or "choppy", it's not incorrect but I've never heard the term. Keep it if it's technical tho.